src/Symfony/Component/TypeInfo/Type/TemplateType.php

```diff
@@ -33,7 +33,7 @@ public function __construct(
 
     public function getBaseType(): BuiltinType|ObjectType
     {
-        throw new LogicException(sprintf('Cannot get base type on "%s" template type.', $this));
```

Comment:
The `Symfony\Component\TypeInfo\Exception\LogicException` use must be removed I guess